Tests/Microsoft.AppCenter.Test.Windows/Utils/UserIdContextTest.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AppCenter.Utils;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using Microsoft.AppCenter.Windows.Shared.Utils;
+
+namespace Microsoft.AppCenter.Test.WindowsDesktop.Utils
+{
+    [TestClass]
+    public class UserIdContextTest
+    {
+        [TestInitialize]
+        public void InitializeUserIdContextTest()
+        {
+           UserIdContext.Instance = new UserIdContext();
+        }
+
+        [TestMethod]
+        public void InstanceIsNotNull()
+        {
+            UserIdContext.Instance = null;
+            Assert.IsNotNull(UserIdContext.Instance);
```

Comment:
You can test singleton too.
Assign it to a local variable, then compare its Assert.Same as the second time you access it.

Reply:
Added test